MovieGallery/wwwroot/css/site.css

```diff
@@ -24,6 +24,10 @@ a {
     border-color: #1861ac;
 }
 
+.img {
+    width: 100%
+}
+
```

Comment:
Added code for img width in the style sheet to remove the warning on Index.cshtml
https://stackoverflow.com/questions/49677911/why-x-is-not-a-valid-value-of-attribute-width-in-visual-studio